### ISSUE TITLE
Daphne v2.2.3 requires that headers be bytes instead of str

### ISSUE THREAD

**Issue by coddingtonbear** [fanout/django-eventstream#24]
Without pinning daphne to `2.2.2`, attempting to connect to the eventstream will raise a exception:

```
Header name 'Content-Type' expected to be `bytes`, but got `<class 'str'>`
```

https://github.com/django/daphne/blob/5722d4e7ea7b40952ffb00e014d7947da89999a2/CHANGELOG.txt#L4

**Comment by jkarneges** [comment]
Thanks for the report! Will fix this shortly.

**Comment by jkarneges** [comment]
Fixed in d57fffabcdc1938da969edbae1c1d83edec670c3
